### PR TITLE
Documentation redirects

### PR DESCRIPTION
**Pull request by toir427**
documentation files redirect with each other

### CODE REVIEW

**Comment by samdark** [comment]
@toir427 what is the problem you are trying to fix with the pull request?

**Comment by toir427** [comment]
I have fixed links moving from one file to the target place of another one. The documentation file works very well in the live version but not working locally. It is useful for the local version.

**Comment by toir427** [comment]
@samdark I wrote this problem your account in telegram

This is not working in almost all. If we remove the span tag after headers and we write a link like this 

```
file1.md
## Querying Data
some text
```
```
file2.md
[querying data](file1.md#querying-data)
```
It is worked

**Comment by toir427** [comment]
Sorry in advance. If I am doing something wrong.

**Comment by bizley** [comment]
No, this is ok, md's headers are now automatically creating ids (anchors), no need to add them manually.

**Comment by toir427** [comment]
Ok

**Comment by bizley** [comment]
No, no, I mean that your PR is ok...

**Comment by toir427** [comment]
I thought it was unnecessary.

File: docs/guide/tutorial-yii-integration.md [hunk 1]
```diff
@@ -118,8 +118,7 @@ Now you can use most features provided by Yii. For example, you can create Activ
 to work with databases.
 
 
-Using Yii 2 with Yii 1 <span id="using-both-yii2-yii1"></span>
-----------------------
+## Using both Yii2 with Yii
```
Comment (samdark) [inline docs/guide/tutorial-yii-integration.md:121]: Previous title was correct. New is not.

File: docs/guide/db-dao.md [hunk 2]
```diff
@@ -638,7 +638,7 @@ $rows = Yii::$app->db->useMaster(function ($db) {
 You may also directly set `Yii::$app->db->enableSlaves` to be `false` to direct all queries to the master connection.
 
 
-## Working with Database Schema <span id="database-schema"></span>
```
Comment (samdark) [inline docs/guide/db-dao.md:641]: That would change the anchor so all the existing links will be broken because of the change. That's probably OK but...

**Review by samdark — CHANGES_REQUESTED** [review]

File: docs/guide/db-dao.md (hunk 2 above)
Comment (toir427) [reply docs/guide/db-dao.md:641]: Why? It is not working very well in the current offline version. Open my [forked repo](https://github.com/toir427/yii2/blob/documentation/docs/guide/tutorial-yii-integration.md#using-both-yii2-with-yii) and check that line. It is worked very well offline and online. And there is also an anchor.

Comment (toir427) [reply docs/guide/db-dao.md:641]: `<span>` tag disrupting for find link

Comment (samdark) [reply docs/guide/db-dao.md:641]: OK. Nothing to break then :)

File: docs/guide/tutorial-yii-integration.md (hunk 1 above)
Comment (toir427) [reply docs/guide/tutorial-yii-integration.md:121]: Sorry for the grammatical mistakes

**Comment by samdark** [comment]
Thank you!

**Comment by softark** [comment]
Ah, just a moment, please.

Manually added anchors have some meaning to document translators.

The original English version may not need them, because in most cases they will be the same with the automatically generated ones. But the things are different for the other languages. In a translated document, you may not want to depend on the auto-generated (i.e., **translated**) anchors. In fact, I prefer adding **English** anchors manually in Japanese translation. I don't think that the auto-generated **Japanese** anchors are more user friendly. When you use English anchors in a translation, the reader will have easier access to the original English version.

So, I would rather want you to keep those manually added anchors in the English documents.

**Comment by bizley** [comment]
Arrghh... You are right, totally forgot about other languages. This needs to be reverted...

**Comment by samdark** [comment]
Reverted.

**Pull request merged**